pcdsdevices/slits.py

```diff
@@ -62,6 +62,42 @@ def __init__(self, *args, nominal_aperture=5.0, **kwargs):
         super().__init__(*args, **kwargs)
         self.nominal_aperture.put(nominal_aperture)
 
+    def format_status_info(self, status_info):
+        """
+        Override status info handler to render the slits.
+
+        Display slits status info in the ipython terminal.
+
+        Parameters
+        ----------
+        status_info: dict
+            Nested dictionary. Each level has keys name, kind, and is_device.
+            If is_device is True, subdevice dictionaries may follow. Otherwise,
+            the only other key in the dictionary will be value.
+
+        Returns
+        -------
+        status: str
+            Formatted string with all relevant status information.
+        """
+        hutch = self.prefix.split(':')[0].upper()
+        stand = self.prefix.split(':')[1].upper()
+        name = f'{hutch} Slit {self.name} on {stand}'
+
+        x_width = status_info.get('xwidth', {}).get('position', 'N/A')
+        y_width = status_info.get('ywidth', {}).get('position', 'N/A')
+        x_center = status_info.get('xcenter', {}).get('position', 'N/A')
+        y_center = status_info.get('ycenter', {}).get('position', 'N/A')
+        w_units = status_info.get('ywidth', {}).get('setpoint', {}).get(
+                                  'units', 'N/A')
+        c_units = status_info.get('ycenter', {}).get('setpoint', {}).get(
```

Comment:
Was thinking a bit about this - since we're going to have hundreds of lines similar to this in these status PRs, what do you think about the following pattern with a helper function?

```python
def get_status_value(status_info, *keys, default_value='N/A'):
   ...

c_units = get_status_value(status_info, 'ycenter', 'setpoint', 'units')
```

Reply:
This is a great idea - thank you, I implemented it in the `BaseInterface` class, but I am wondering if I should just place it in `utils.py` ?

Reply:
Glad you like it - `utils.py` is just fine, I think!

Reply:
Moved it to `utils` in 1babb1c - but please let me know if there is a better place for this one.
Also, I found this idea online of using the `operator.getitem` and `functools.reduce` - which I have not used before but I think they work great here.... needs some testing.

Reply:
Good call on the functional tools. Only thing it seems you might need is an `except KeyError: return default_value`

Reply:
Yes that's right, just added that in 8e01f24 :)